src/open_inwoner/templates/pages/plans/create-no-template.html

```diff
@@ -0,0 +1,99 @@
+{% extends 'master.html' %}
+{% load i18n button_tags card_tags map_tags form_tags grid_tags icon_tags string_tags link_tags %}
+
+{% block content %}
+    <div class="plan-create">
+        <div class="plan-template__header">
+            <h1 class="utrecht-heading-1">{% trans "Start een nieuwe samenwerking" %}</h1>
+            <div class="plan-template__header-nav">
+                {% if request.user.get_active_contacts %}
+                    {% button href="#plan-templates" text=_("Gebruik sjabloon") primary=True transparent=True extra_classes="link--bold" icon="insert_drive_file" icon_outlined=True %}
+                    {# TODO: Change scrollbutton into "href="collaborate:plan_choose_template" #}
+                {% endif %}
+            </div>
+        </div>
+
+        {% optional_paragraph configurable_text.plans_page.plans_intro %}
+
+        {% render_grid %}
+            {% render_column start=4 span=6 %}
+                {% if request.user.get_active_contacts %}
+                    {% render_form id="plan-form" form=form method="POST" %}
+                        {% csrf_token %}
+                        {% input form.title %}
+                        {% input form.goal %}
+                        {% input form.description %}
+                        
+{#                        TODO: create new model field to add a File to the plan #}
+{#                      {% input form.file %} #}
+                        
+                        {% date_field form.end_date icon="calendar_today" icon_position="after" %}
+                        <div class="plan__contacts">
+                            <p class="utrecht-paragraph">Contactpersoon</p>
+                            {% checkboxes form.plan_contacts %}
+                        </div>
+
+                        <div class="">
+                            <div class="plan-template" id="plan-templates">
+                                <div class="label plan-template__row plan-template__table radio">
+                                    <div>
+                                        <input class="radio__input" type="radio" name="template" value="" id="id_template_0" {% if not form.data.template %}checked{% endif %}>
+                                        <label class="radio__label" for="id_template_0">Geen template</label>
+                                    </div>
+                                </div>
+                                {% for tpl_id, plan_template in form.template.field.choices %}
```

Comment:
Do we still want to display the template choices here? I think it would make more sense to add a link to the template choice page. This is relevant for how we want to deal with the forms to make this functional (see general comment).

Reply:
These will indeed need to be removed for the "no-template" option - I wanted to keep 1 functional page before merge with functional URL, but I guess I should just have kept `create.html`,
and should have added `create-no-template.html `without any link to it, then later give that new no-template page the same URL as the current `.../create` page.